clang-tools-extra/docs/clang-tidy/checks/bugprone/reset-call.rst

```diff
@@ -0,0 +1,33 @@
+.. title:: clang-tidy - bugprone-reset-call
+
+bugprone-reset-call
+===================
+
+Finds calls to ``reset()`` method on smart pointers where the pointee type
```

Comment:
Please make this statement same as statement in Release Notes.

Reply:
Done, also removed parentheses in ``reset`` to follow general style